packages/ListBoxWithTags/src/ListBoxWithTags.js

```diff
@@ -76,60 +36,65 @@ const renderTrigger = ({
     onRemove(option);
   };
 
-  function renderCaret() {
-    return isOpen ? <triggerSc.UpIcon /> : <triggerSc.DownIcon />;
+  function renderCaret(isDisabled) {
+    return isOpen ? <sc.UpIcon isDisabled={isDisabled} /> : <sc.DownIcon isDisabled={isDisabled} />;
   }
 
+  const triggerProps = propsForTrigger();
   const a11yTextOptions = selectedOptions.map(item => {
     return tagLabelKey === null ? item.label : item[tagLabelKey];
   });
-
   const a11yText = selectedOptions.length === 0 ? t("listBoxWithTags.placeholder") : a11yTextOptions.join(", ");
 
   return (
-    <sc.Trigger
-      a11yText={t("listBoxWithTags.a11y_text_trigger", { options: a11yText })}
-      aria-controls={listBoxId}
-      aria-expanded={isOpen}
-      aria-haspopup
-      ref={refTrigger}
-      {...propsForTrigger()}
-      onClick={handleClick}
-      onKeyUp={handleKeyUp}
-      onKeyDown={handleKeyDown}
-      size={size}
-      allOptionsAreSelected={allOptionsAreSelected}
-    >
-      <Tags>
-        {selectedOptions.map(item => {
-          if (typeof renderTag === "function") {
-            return renderTag({ option: item, Tag, onRemove: handleRemove(item) });
-          }
-
-          const label = tagLabelKey === null ? item.label : item[tagLabelKey];
-
-          if (typeof label !== "string") {
-            throw Error(
-              `Your item ${JSON.stringify(
-                item
-              )} must include the attribute "label", or you must indicate which attribute should be rendered as the label via the "tagLabelKey" prop.`
+    <>
+      <sc.Trigger
+        {...triggerProps}
+        aria-controls={idListBoxContent}
```

Comment:
`triggerProps` also includes `aria-controls`, but it assumes we use the same `idListBox` for the `id` of the controlled content, which we don't, since we don't have that value when we assign `idListBoxContent`.